packages/page-pattern-modal/src/components/page-pattern-modal.tsx

```diff
@@ -297,6 +309,10 @@ class PagePatternModal extends Component< PagePatternModalProps, PagePatternModa
 			<Modal
 				title="" // We're providing the title with the `aria.labelledby` prop
 				className="page-pattern-modal"
+				/* eslist-disable-next-line @typescript-eslint/ban-ts-comment */
+				// @ts-ignore `onRequestClose`'s type is () => void but ideally it should eslint-disable-line
```

Comment:
`eslint-disable-line` slipped through the cracks here when I was testing. I'll remove it.

Reply:
We could also use `@ts-expect-error` to avoid having to disable the eslint rule. (I think it's recommended for when you are able to follow-up and fix the error.)

Reply:
Ah, I didn't know about this one, nice!

Reply:
Just learned about it today!